utils/mixins/singleSubscribeMixin.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Vue } from 'nuxt-property-decorator'
+// declare type UnsubscribePromise = Promise<Unsubscribe>;
+declare type Unsubscribe = () => void
+
+/*
+ * refer to https://stackoverflow.com/questions/51873087/unable-to-use-mixins-in-vue-with-typescript
+ * usage import Component, { mixins } from 'vue-class-component';
+ * class ExtendedClass extends mixins(SubscribeMixin) {
+ */
+@Component
+export default class SingleSubscribeMixin extends Vue {
+  private sub: Unsubscribe = () => void 0
```

Comment:
`void 0`?

Reply:
> why and what this new mixin is for

The use-case is that @chandradot99 has to subscribe for the user's balance.
We also need to correctly handle when the user logs in under different address